test/integration_test.py

```diff
@@ -235,7 +235,7 @@ def _list_partition_bundles(self,
         else:
             expected_partition_size = 16
         lower = expected_partition_size / 2
-        upper = expected_partition_size * 1.5
+        upper = expected_partition_size * 2
```

Comment:
Smallest number that worked without going to an unreasonable number of decimal places.